README.md

```diff
@@ -18,6 +18,13 @@ Create a new project of your own from this template repository, through the butt
 ### Cleanup
 Make any file additions/modifications you want, such as removing solution files for previous years if you've no interest in completing those. You probably do not want to remove any files outside of `AdventOfCode/Solutions/` unless you know what you're doing.
 
+### Generating Previous Year's Solution Files
+Use the included PowerShell script `AdventOfCode/GenerateSolutionFiles.ps1` to generate a year's solution files following the same layout as those aleardy included.
```

Comment:
Minor typo `aleardy` on this line that I'd prefer be fixed.

Reply:
Done in 1126f4b